README.md

```diff
@@ -1,221 +1,172 @@
+There are some buttons associated with the Explorer that will perform actions on the file-system. Three buttons are located on the header and are used for adding, saving and modifying the current file (respectively). The modify button will open a modal in which the file can be renamed or deleted. The four buttons above the Explorer in Editor mode are used for adding, deleting and modifying a file as well as refreshing the Explorer (handy when files have changed on the file-system). A file or folder can be added to  by selecting the folder first then pressing the add button.
 
-You can select a configuration by clicking on it. The flow will be loaded on the white canvas. Two additional options will be provided by right-clicking a file: Rename and delete.
+You can select a configuration by clicking on it. The Flow will be loaded on the Canvas.
 
 ### 🎨 Palette
 
-The palette contains all the usable pipes and validators out of which the adapters are build. The pipes can be searched and are categorized respectively.
-
-The pipe will be added to the canvas by clicking on it.
-
-### 🔌 Flow
+The palette contains all the usable listeners, pipes, validators and an exit out of which the adapters are build. These elements can be searched and are categorized respectively.
 
-The flow is the main attraction of the application and has a lot of hidden features. There are a number of ways to manipulate the flow:
+The element will be added to the canvas by clicking on it.
 
-- You can move the pipes by dragging them around.
-- A forward can be created by dragging from the edge of the first pipe, to the edge of the second pipe. While dragging you can see an arrow, pointing in the direction of the forward.
+### 🏗️Flow
 
-Besides dragging and clicking the pipes, there are some additional options in the right-click menu:
+The Flow is the main attraction of the application and has a lot of hidden features. There are a number of ways to manipulate the Flow:
 
-- Toggle activity mode: The pipes will change to a simpler and compacter appearance by clicking this option.
-- Realign flow: You can regenerate the locations of the pipes (without changing the run-time behavior of your configuration).
-- Toggle curve: Change the angled lines to curves.
-- Toggle flow direction: Change to flow from vertical to horizontal mode.
-- Export SVG: This option will export an SVG image of the part of the canvas you are looking at.
+- You can move the elements by dragging them around.
+- A forward can be created by dragging from the green circle of the first element, to the yellow circle of the second element. While dragging you can see an arrow, pointing in the direction of the forward.
+- A forward can be removed by double clicking it or by disconnecting it from the circle. The latter can be tricky if there are a lot of forwards connected to an element.
```

Comment:
Removing a forward by double-clicking it does not work

Reply:
You need to be very specific for it to work

Reply:
You mean clicking at a very specific spot before double-clicking?

Reply:
The clicking needs to be pixel pres ice. This isn't very user-friendly and will be changed in the issue "Improve ux for removing forwards" (#220).

I think the text is good for now.